examples/live/run.sh

```diff
@@ -10,49 +10,91 @@ export HDF5_USE_FILE_LOCKING="FALSE"
 gps_start_time=1272790000
 gps_end_time=1272790500
 
-echo -e "\\n\\n>> [`date`] Making template bank"
-
-curl \
-    --remote-name \
-    --silent \
-    --show-error \
-    https://raw.githubusercontent.com/gwastro/pycbc-config/710dbfd3590bd93d7679d7822da59fcb6b6fac0f/O2/bank/H1L1-HYPERBANK_SEOBNRv4v2_VARFLOW_THORNE-1163174417-604800.xml.gz
-
-pycbc_coinc_bank2hdf \
-    --bank-file H1L1-HYPERBANK_SEOBNRv4v2_VARFLOW_THORNE-1163174417-604800.xml.gz \
-    --output-file template_bank_full.hdf
-
-rm -f H1L1-HYPERBANK_SEOBNRv4v2_VARFLOW_THORNE-1163174417-604800.xml.gz
-
-pycbc_hdf5_splitbank \
-    --bank-file template_bank_full.hdf \
-    --output-prefix template_bank_ \
-    --random-sort \
-    --random-seed 831486 \
-    --templates-per-bank 1000
-
-mv template_bank_0.hdf template_bank.hdf
-rm -f template_bank_*.hdf
-
-echo -e "\\n\\n>> [`date`] Generating simulated strain"
-
-function simulate_strain { # detector PSD_model random_seed
-    mkdir -p strain/$1
-    pycbc_condition_strain \
-        --fake-strain $2 \
-        --fake-strain-seed $3 \
-        --output-strain-file "strain/$1/$1-SIMULATED_STRAIN-{start}-{duration}.gwf" \
-        --gps-start-time $gps_start_time \
-        --gps-end-time $gps_end_time \
-        --sample-rate 16384 \
-        --low-frequency-cutoff 10 \
-        --channel-name $1:SIMULATED_STRAIN \
-        --frame-duration 32
-}
-
-simulate_strain H1 aLIGOMidLowSensitivityP1200087 1234
-simulate_strain L1 aLIGOMidLowSensitivityP1200087 2345
-simulate_strain V1 AdVEarlyLowSensitivityP1200087 3456
+# test if there is a template bank. If not, make one
+
+if [[ ! -f template_bank.hdf ]]
+then
+    echo -e "\\n\\n>> [`date`] Making template bank"
+    curl \
+        --remote-name \
+        --silent \
+        --show-error \
+        https://raw.githubusercontent.com/gwastro/pycbc-config/710dbfd3590bd93d7679d7822da59fcb6b6fac0f/O2/bank/H1L1-HYPERBANK_SEOBNRv4v2_VARFLOW_THORNE-1163174417-604800.xml.gz
+
+    pycbc_coinc_bank2hdf \
+        --bank-file H1L1-HYPERBANK_SEOBNRv4v2_VARFLOW_THORNE-1163174417-604800.xml.gz \
+        --output-file template_bank_full.hdf
+
+    rm -f H1L1-HYPERBANK_SEOBNRv4v2_VARFLOW_THORNE-1163174417-604800.xml.gz
+
+    pycbc_hdf5_splitbank \
+        --bank-file template_bank_full.hdf \
+        --output-prefix template_bank_ \
+        --random-sort \
+        --random-seed 831486 \
+        --templates-per-bank 50
+
+    mv template_bank_0.hdf template_bank.hdf
+    rm -f template_bank_*.hdf
+fi
+
+# test if there is a hwinj file. If not, make one.
+# if a new inj is made, delete old strain and output files
+
+
+
+
+if [[ "$(echo ./hwinjcbc*.xml.gz)" != "./hwinjcbc*.xml.gz" ]]
+then
+    inj_names='ls ./hwinjcbc*.xml.gz'
+    n_inj=${#inj_names[@]}
+    if [[ $n_inj = 1  ]]
+    then echo -e "\\n\\n>> [`date`] Pre-existing Inj Xml Found"
+    elif [[ $n_inj > 1 ]]
+    then echo -e "\\n\\n>> [`date`] Too many injection xmls, only first will be used"
```

Comment:
What motivates this check? We are never going to generate more than one injection file.

Reply:
The code is currently set up that it can take an injection xml as input instead of generating it's own by checking for a pre-existing xml injection file. I just wanted to make it clear that if for some reason there are two xml injection files sitting in the directory only one will be used.

Reply:
Went ahead and made the suggested changes, checking for too many injection files will be a feature for a larger non-travis test in the future.